app/helpers/users_helper.rb

```diff
@@ -58,4 +58,8 @@ def users_tags_gradation(count, max_count)
       'is-mid'
     end
   end
+
+  def user_practices(user)
+    user.course.categories.order(:position).map(&:practices).flatten
```

Comment:
[flat_map](https://docs.ruby-lang.org/ja/latest/method/Enumerable/i/collect_concat.html) の方が少し短くなる ⛳ 

```suggestion
    user.course.categories.order(:position).flat_map(&:practices)
```